src/components/LocationSelector/LocationSelectorDialog/__tests__/LocationSelectorDialog.spec.tsx

```diff
@@ -11,24 +13,18 @@ describe('LocationSelectorDialog component', () => {
     const selectedLocations = [
```

Comment:
maybe it is easier/cleaner to do `const selectedLocations: LocationSelectorLocation[] = [...]` instead of casting every item.

Reply:
When i try to do it. It shows error. Also I've checked other places and we do cast there not only tests
![image](https://github.com/textkernel/oneui/assets/104063210/13b0b255-26bf-46a3-a365-7c34f9217408)